.gitignore

```diff
@@ -27,6 +27,7 @@
 .local-version
 .libs
 .vagrant
+.vscode
```

Comment:
```suggestion
.vscode/
```

Reply:
I am following the style above this line. @jwhui @simonlingoogle Do you think it is necessary to change them all?
```
.deps
.dirstamp
.DS_Store
.local-version
.libs
.vagrant
.vscode
```

Reply:
I think we should move .vscode/ below (together with .idea/) because they are both IDE configuration files.